src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -411,3 +415,122 @@ def get(self, request: Request, project: Project, credentials_id: str) -> Respon
             },
             status=200,
         )
+
+
+class AppStoreConnectStatusEndpoint(ProjectEndpoint):  # type: ignore
+    """Returns a summary of the project's App Store Connect configuration
+    and builds.
+
+    ``GET projects/{org_slug}/{proj_slug}/appstoreconnect/status``
+
+    Response:
+    ```json
+    [
+        {
+            "id": "abc123",
```

Comment:
since we get the id as a property, do we need the id inside of the object too?

ex:

```
{
 "4e1c91f46aea4101b2953192d61cef4b": {
      "id":"4e1c91f46aea4101b2953192d61cef4b",
      "credentials": ...
    }
}
 
```

Reply:
good point, i'll see what i can do to remove that.

Reply:
done 🎉